packages/host/app/services/realm-info-service.ts

```diff
@@ -56,4 +66,16 @@ export default class RealmInfoService extends Service {
       return realmInfo;
     }
   }
+
+  fetchAllKnownRealmInfos = restartableTask(async () => {
+    let paths = [...new Set([ownRealmURL, ...otherRealmURLs])].map(
```

Comment:
does this include the base realm?

Reply:
I removed base realm url because this seems to include it. But I suppose it might not always be included... I'll add it back.